epoxy-processor/src/main/java/com/airbnb/epoxy/ClassToGenerateInfo.java

```diff
@@ -134,4 +170,16 @@ public ConstructorInfo(Set<Modifier> modifiers, List<ParameterSpec> params) {
       this.params = params;
     }
   }
+
+  public static class MethodInfo {
```

Comment:
I dunno, don't think it's a good idea because we are building actual methods later, so we need this intermediate data, so we don't mix this logic with code generation logic.